backend/routers/referral.py

```diff
@@ -0,0 +1,44 @@
+from fastapi import APIRouter, Depends, Header, HTTPException
+from models.database import SessionLocal
+from sqlalchemy.orm import Session, joinedload
+import models.referral as model
+import schemas.referral as schema
+
+router = APIRouter()
+
+def get_db():
+    db = SessionLocal()
+    try:
+        yield db
+    finally:
+        db.close()
+
+@router.get("/referrals")
+async def get_prescription(patient_id = Header(1), db: Session = Depends(get_db)):
+    response = db.query(model.Referral).filter(model.Referral.patient_id == patient_id).all()
+    return response
+    
+@router.get("/referals/{referal_id}")
```

Comment:
correct url to be "/referrals/{referral_id}"

Reply:
Yup